zebra-state/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zebra-state"
-version = "3.0.0"
+version = "3.0.1"
```

Comment:
Following the same logic, this probably requires a minor bump.

Reply:
Bumped minor version in https://github.com/ZcashFoundation/zebra/pull/10088/commits/7c36471ad9aef254b85a566c171422c084fa2bf3.